### PR TITLE
[No ready to Merge] Readme update

### PR DESCRIPTION
**Pull request by Baaaaam**
Hi,

This is an illustration of the discussion I [started on Cyclus dev](https://groups.google.com/forum/#!topic/cyclus-dev/3P6VxBkQ-Xw)
I have been rearranging the README, to simplify it.... This is not final but give a good overall view of what I am attempting to do!

This is how I propose to modify the flow inside the README:

1. Brief dependency description with command line to install using more common package manager (apt-get, port...) (conda ?)
2. mention other installation process link to INSTALL.rst and website (which should contain the same information) for a complete description of all the installation procedure of the dependency: VM, conda, ubuntu (from source + deb package), MacOS
3. complete description of Cyclus installation from `compilation` to `unit_test`
4. Description of the Workflow used in the Cyclus/Cycamore git repo (maybe put the example on a link)

My goal is to simplify the README at the maximum to allow potential developer to get the information they need: dependency, cyclus compilation and workflows.
I feel like the actual README is to big: the information are important, but probably not required there.... We should just get link to get all those detailed informations...

My goal is to transfer most part of the installation explanation to the INSTALL.rst file and the fuel cycle.org website...

Plz give me feedback about what you think...

Baptiste

### CODE REVIEW

File: INSTALL.rst [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+#. Login with the username ``ubuntu`` and pw ``reverse``
+#. Skip to "Installing Cyclus"
+
+Note that the Cyclus source code exists already on the virtual machine. You can
+update it with
+
+.. code-block:: console
+
+    $ cd cyclus
+	$ git pull origin cyclus
```
Comment (scopatz) [inline INSTALL.rst]: Spacing.

**Comment by scopatz** [comment]
+1 from me.

**Comment by gonuke** [comment]
I think both the README and the INSTALL might benefit from a table of contents.  I'm pretty sure there is no way to have it done automatically by GH.  (There are scripts out there that one can run on the README file to add it, but let's not go there yet.)

**Comment by scopatz** [comment]
yeah you have to write these yourself

**Comment by Baaaaam** [comment]
@gonuke @scopatz @FlanFlanagan @mbmcgarry @rwcarlsen @anyone
 I start some work there, this is not finish, but could you tell me what you think about it.

My goal is to do a bunch of with a simple dedicated goal, to help the different users to find the information they need:
- a README with simple installation (install.py), run test, run a simple input
- a INSTALL file with a complete explanation of how to install cyclus from source
- a DEPENDENCY file: how to install dependency
- a workflow file: how to contributed to Cyclus

My plan is to put all those information on the website also, + some additional installation method:
conda, deb package, which are not necessary dedicated to programmer user...

my goal is to allow:
-  "click" user to find to proper installation guide without any additional confusion information, 
- "medium skilled user" to find some information to install dependancies, and run the install script (whit some customization), cyclus git workflow
- "skilled user" to get directly the information they need: require dependencies, install path and cyclus git workflow

Please don't waste to much time correcting my english now... I am trying to set a proper global shape before going into the detail of each part...

**Comment by mbmcgarry** [comment]
General thoughts:
- Somewhere in the beginning we need to note this doesn't run on PC and requires a terminal.  Otherwise people will skip to "Quick Cyclus Installation" and then get stuck.
- Quick Cyclus Installation assumes you've already downloaded the github repository without saying this explicitly.
- Many places have a pair of links, eg "For detailed installation instruction, visit the Cyclus Website or INSTALLATION guide". Wherever possible we should only provide One link for specific info, otherwise people won't know which one is the one they really need.

Comments regarding sophistication of 'Click Users' and 'Medium Skill Users':
- They will likely assume that they don't need to worry about this 'dependencies' section and just try to skip it.
- They may not know how to edit their $PATH

 I like the idea of making this information modular, but the dependencies page will likely be very intimidating to non-expert users.  I think there is value in having the explicit one-line dependency installation for Linux (using apt-get) either on the main page or on the Installation Instructions.

It may be worth considering other modular organization schemes. One idea: have the front page have choices by skill level "Non-programmer, new-programmer, experienced-progammer" (or similar) and then have each link give full start-to-finish directions based on skill. Another idea:  choices by platform (Linux vs Mac vs PC w/Virtual Box).

**Comment by gonuke** [comment]
Maybe we need a 1/2 day documentation hack-a-thon? or at least a meeting to discuss the strategies.

**Comment by Baaaaam** [comment]
Should I continu it or wait for a discussion ?

**Comment by Baaaaam** [comment]
I close this as we decided to do otherwise... (I don't remove my branch in case of we want to take some back...)

**Pull request closed without merging**